package.json

```diff
@@ -13,7 +13,10 @@
     "lint:all": "nx run-many --all --target=lint",
     "e2e:all": "nx run-many --all --target=e2e",
     "vegacapsule": "vegacapsule network bootstrap --config-path=../frontend-monorepo/vegacapsule/config.hcl",
-    "release": "git checkout develop ; git pull ; node scripts/make-release.js"
+    "release": "git checkout develop ; git pull ; node scripts/make-release.js",
+    "install:console-test": "cd apps/console-test && poetry install && poetry run playwright install chromium && python -m vega_sim.tools.load_binaries --force --version v0.73.1",
```

Comment:
Can we make these commands consistent and for now can we just name them the same as the app directory I'd suggest

Maybe
```
trading:test
trading:test:install
trading:test:all
```

Reply:
Also is there anyway we can check that poetry is running, perhaps we do this in the top level fixture?